src/state/apis/zapper/zapperApi.ts

```diff
@@ -60,6 +62,28 @@ export const zapperApi = createApi({
           }),
         )
 
+        const zapperAssets = zerionV2AppTokensData.reduce<AssetsState>(
+          (acc, appTokenData) => {
+            const assetId = toAssetId({
```

Comment:
we might want to early return if we already have these assets in state, as we're likely to have richer data (colors/icons) in pregenerated stuff

Reply:
As discussed, we might want to maybe do this in another diff once we are more familiar with the heuristics of this, currently ETH/FOX is the only UNI-V2 asset we "fetch" (hardcode in lib really) and using it instead of the zapper data would mean we would be inconsistent